config/tomlConfig_test.go

```diff
@@ -775,6 +775,9 @@ func TestEnableEpochConfig(t *testing.T) {
     # FixDelegationChangeOwnerOnAccountEnableEpoch represents the epoch when the fix for the delegation system smart contract is enabled
     FixDelegationChangeOwnerOnAccountEnableEpoch = 87
 
+	# ScToScLogEventEnableEpoch represents the epoch when the sc to sc log event feature is enabled
+	ScToScLogEventEnableEpoch = 88
```

Comment:
missing definition in cmd/node/config/enableEpochs.toml

Reply:
done